modules/light-clients/07-tendermint/store.go

```diff
@@ -280,7 +280,7 @@ func PruneAllExpiredConsensusStates(
 ) int {
 	var heights []exported.Height
 
-	pruneCb := func(height exported.Height) bool {
+	pruneCbFn := func(height exported.Height) bool {
```

Comment:
I don't think we should enforce using the `Fn` suffix. I don't think it provides too much value, and for a lot of cases its usually pretty clear from naming that its a function. E.g. here we're using `cb` for callback, where a callback is always a function. Similarly in the ante handler `next` is standard naming conventions for middleware patterns like the ante handler chain so I think `nextFn` doesn't do much. 

That's just my opinion anyways, I would prefer to not be super strict about this style choice! :)

Reply:
Sounds good to me, @damiannolan. I will change it.